cmd/operator-sdk/main.go

```diff
@@ -57,7 +57,8 @@ func main() {
 				log.Debug("Debug logging is set")
 			}
 			if err := checkDepManagerForCmd(cmd); err != nil {
-				log.Fatal(err)
+				// Allow the cases of some sub-commands might not require use the deps files. 
+				log.Warnf("Operator type %s Go and the %s. Please, review the structure of your project.", projutil.GetOperatorType(), err)
```

Comment:
I'd change the comment to say something like:
`// Allow cases where subcommands do not require deps files`
Also the log warning is confusing to me. Let's assume `projutil.GetOperatorType()` returns `go`. Let's also assume the `err` is not nil and returns the message `error text`. The message will look as follows:

 "Operator type go Go and the error text. Please, review the structure of your project."

If I saw this error, I honestly don't know what it is trying to tell me. Can you clarify what your intend to convey to the user?

Reply:
Good catch. Fixed 👍